tests/integration/test_reload.py

```diff
@@ -146,3 +146,71 @@ async def test_reload_failure(
 
         # the config should be unchanged
         assert schd.config.cfg['scheduling']['graph']['R1'] == 'one'
+
+
+async def test_orphan_reload(
+    flow,
+    scheduler,
+    start,
+    log_filter,
+):
+    """Reload should not fail about orphaned tasks.
+
+    The following aspects of reload about orphans are tested:
+        - Removal of both xtrigger and associated active/incomplete task.
+        - Broadcast deltas generated after reload.
```

Comment:
Check that I have the issue mapping the right way around!
```suggestion
        - Removal of both xtrigger and associated active/incomplete task.
          https://github.com/cylc/cylc-flow/issues/6814
        - Broadcast deltas generated after reload.
          https://github.com/cylc/cylc-flow/issues/6815
```